src/Maker/MakeEntity.php

```diff
@@ -97,6 +97,7 @@ public function configureCommand(Command $command, InputConfiguration $inputConf
             ->addOption('broadcast', 'b', InputOption::VALUE_NONE, 'Add the ability to broadcast entity updates using Symfony UX Turbo?')
             ->addOption('regenerate', null, InputOption::VALUE_NONE, 'Instead of adding new fields, simply generate the methods (e.g. getter/setter) for existing fields')
             ->addOption('overwrite', null, InputOption::VALUE_NONE, 'Overwrite any existing getter/setter methods')
+            ->addOption('table-name', 't', InputOption::VALUE_NONE, 'Set custom table name')
```

Comment:
```suggestion
            ->addOption('table-name', 't', InputOption::VALUE_REQUIRED, 'Set custom table name')
```

This is the reason of the test failures I think.

But i'm wondering here... should we add a test to avoid using an existing table name ?